stable/jaeger-operator/templates/service.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "jaeger-operator.fullname" . }}-metrics
+  labels:
+    app.kubernetes.io/name: {{ include "jaeger-operator.name" . }}
```

Comment:
use the {{ include "jaeger-operator.labels" . | indent 4 }}

Reply:
changed